### PR TITLE
fix: fingerprint auto accepted still asks for admin confirmation

### PR DESCRIPTION
**Pull request by debdutdeb**

### CODE REVIEW

File: .changeset/shaggy-yaks-train.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Fixed an issue where Rocket.Chat would ask admins to confirm fingerprint change (new workspace vs configuration update), even with `AUTO_ACCEPT_FINGERPRINT` environment variable set to `"true"`.
```
Comment (debdutdeb) [inline .changeset/shaggy-yaks-train.md:5]:
```suggestion
Fixed an issue where Rocket.Chat would ask admins to confirm fingerprint change (new workspace vs configuration update), even when `AUTO_ACCEPT_FINGERPRINT` environment variable set to `"true"`.
```

**Review by ruru-m07 — APPROVED** [review]

**Review by geekgonecrazy — APPROVED** [review]

**Review by KevLehman — APPROVED** [review]

**Comment by ggazzo** [comment]
/backport. 6.6.7

**Comment by ggazzo** [comment]
/backport. 6.6.7

**Comment by ggazzo** [comment]
/backport. 6.6.7

**Comment by ggazzo** [comment]
/backport. 6.6.7

**Comment by ggazzo** [comment]
/backport. 6.6.7

**Pull request merged**